tasks/create-github-release/tests/test-create-github-release.yaml

```diff
@@ -71,11 +65,5 @@ spec:
                 cat $(workspaces.data.path)/mock_gh.txt
                 exit 1
               fi
-
-              if [ "$(params.url)" != \
```

Comment:
Why are we removing the check that the url result is what we expect?

Reply:
Because we are not using the  `echo` of the release url at the end of the task anymore to send to the results. I found it cleaner to pipe the output of the `gh` command since it outputs the URL that we need. And since in the tests what we are doing is mocking that `gh`  command, I don't see the point of testing an  `echo` that I manually need to create in the `mock.sh`